### PR TITLE
Fix codegen fail for test project and enabled operationIdentifier generation

### PR DESCRIPTION
**Pull request by Codebear98**
1. code gen failed due to the Apollo script line 1. in StarWarAPI and GithubAPI which returns 2 records.
`APOLLO_FRAMEWORK_PATH=$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 1)`
change to 
`APOLLO_FRAMEWORK_PATH=$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 1 -print | head -n 1)`

2. add `--operationIdsPath=operationIdsPath.json` to generate `operationIdentifer`

### CODE REVIEW

File: Tests/GitHubAPI/operationIdsPath [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (designatednerd) [inline Tests/GitHubAPI/operationIdsPath]: Should this file maybe go to `operationIdsPath.json` instead of not having an extension?

File: Tests/GitHubAPI/API.swift [hunk 2]
```diff
@@ -16,23 +18,23 @@ public final class RepositoryQuery: GraphQLQuery {
       GraphQLField("repository", arguments: ["owner": "apollographql", "name": "apollo-ios"], type: .object(Repository.selections)),
     ]
 
-    public var snapshot: Snapshot
+    public private(set) var resultMap: ResultMap
```
Comment (designatednerd) [inline Tests/GitHubAPI/API.swift]: This change doesn't seem to be related to the operation identifiers - do you know why this is coming through here? It does seem like it was already changed in the star wars example

File: Apollo.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -641,6 +642,7 @@
 				9FCE2D171E6C259B00E34457 /* Starship.graphql */,
 				9FCE2D0C1E6C259B00E34457 /* CreateReviewForEpisode.graphql */,
 				9FCE2D0A1E6C258A00E34457 /* API.swift */,
+				F8E9D8AE22B2492C0065DA98 /* schema.json */,
```
Comment (designatednerd) [inline Apollo.xcodeproj/project.pbxproj]: any idea what schema was added here that wasn't already in there?

**Comment by Codebear98** [comment]
> Can you explain for those of us who suck at bash what the `| head -n 1` thing is doing?
Github project is failed to run in master branch (refer to screen capture). I guess it's been awhile.

Screenshot 2019-07-12 at 12 22 38 PM

The failure reason is the original script accept multiple records as search result
`APOLLO_FRAMEWORK_PATH=$(eval find $FRAMEWORK_SEARCH_PATHS -name "Apollo.framework" -maxdepth 1)`
```

To debug I added a print out as below
`echo "Debug[APOLLO_FRAMEWORK_PATH] $APOLLO_FRAMEWORK_PATH [End]"`

```
:-1: Debug[APOLLO_FRAMEWORK_PATH] /Users/henry.hong/Library/Developer/Xcode/DerivedData/Apollo-feshfmhssebrsidctwtelmmgfodm/Build/Products/Debug-iphonesimulator/Apollo.framework

:-1: /Users/henry.hong/Library/Developer/Xcode/DerivedData/Apollo-feshfmhssebrsidctwtelmmgfodm/Build/Products/Debug-iphonesimulator/Apollo.framework [End]
```

Solution is only accept first search result by adding ` -print | head -n 1`

File: Apollo.xcodeproj/project.pbxproj (hunk 3 above)
Comment (Codebear98) [reply Apollo.xcodeproj/project.pbxproj]: This is schema.json in Starwar folder. 
Because schema.json in Github folder was also added to the project.
Just sync up the practice by also add this Starwar schema.json to the project.

File: Tests/GitHubAPI/API.swift (hunk 2 above)
Comment (Codebear98) [reply Tests/GitHubAPI/API.swift]: Yes, you are right. It's not related to operation identifiers.
The changes is related to the fix of CodeGen. Once CodeGen is running successfully, this files updated. I guess GitHubAPI have been failure for awhile which made this file outdated.

File: Tests/GitHubAPI/operationIdsPath (hunk 1 above)
Comment (Codebear98) [reply Tests/GitHubAPI/operationIdsPath]: Agreed! Updated, thanks!

File: Tests/GitHubAPI/API.swift (hunk 2 above)
Comment (designatednerd) [reply Tests/GitHubAPI/API.swift]: OK thanks!

File: Apollo.xcodeproj/project.pbxproj (hunk 3 above)
Comment (designatednerd) [reply Apollo.xcodeproj/project.pbxproj]: ahhhh got it, thanks!

**Comment by designatednerd** [comment]
@Codebear98 AHA! This helped me figure out why a script that was working on external projects wasn't working in the direct project, and I got that opened as #629. I think once that's merged we can probably scrap this PR and concentrate on #608 directly since it'd bring the changes down pretty significantly to the things that are around APQ generation

Thank you very much for helping figure this out!

**Comment by designatednerd** [comment]
OK! I think if you rebase #608 I should be able to take a look at that on Monday, and you can close this one out. Thanks for your patience and have a nice weekend!

**Comment by Codebear98** [comment]
Ok, thanks for your review!
I’ll be working on #608 on Monday!
Have a nice day!
